modules/messages/client/api/messages.api.js

```diff
@@ -15,8 +15,49 @@ export async function fetchThreads(params = {}) {
       nextParams = params;
     }
   }
+  return {
+    data,
+    nextParams,
+  };
+}
+
+export async function fetchThreads(params = {}) {
+  const { data: threads, nextParams } = await fetchWithNextParams(
+    '/api/messages',
+    params,
+  );
   return {
     threads,
     nextParams,
   };
 }
+
+export async function fetchMessages(userId, params = {}) {
+  const { data: messages, nextParams } = await fetchWithNextParams(
+    `/api/messages/${userId}`,
+    params,
+  );
+  return {
+    messages,
+    nextParams,
+  };
+}
+
+export async function sendMessage(userToId, content) {
+  const { data: message } = await axios.post('/api/messages', {
+    userTo: userToId,
+    content,
+    read: false,
+  });
+  return message;
+}
+
+export async function markRead(messageIds) {
+  // the server response is a bit wrong, status 200, but no body or content-type, so we
```

Comment:
Can we change the response from server? :-)

Reply:
Good point! Done.